scripts/generate_pinval/generate_pinval.py

```diff
@@ -165,7 +187,13 @@ def build_front_matter(
         "pin_pretty": pin_pretty(tp["meta_pin"]),
         "pred_pin_final_fmv_round": f"${tp['pred_pin_final_fmv_round']:,.2f}",
         "cards": [],
-        "var_labels": {k: pretty_fn(k) for k in preds_cleaned},
+        "var_labels": {
+            **{k: pretty_fn(k) for k in preds_cleaned},
+            # These have no entries in our vars_dict, so we add them manually
+            "pin": "PIN",
+            "document_num": "Sale Doc. Num.",
+            "property_address": "Address",
+        },
```

Comment:
Main constraint here is that we couldn't use the vars_dict to transform these since these columns aren't in the underlying csv file.